recipe/install_nvcc.sh

```diff
@@ -65,8 +65,13 @@ fi
 
 if [[ -z "\$(\${CUDA_HOME}/bin/nvcc --version | grep "Cuda compilation tools, release ${PKG_VERSION}")" ]]
 then
-  echo "Version of installed CUDA didn't match package"
-  return 1
+  if [[ "\${CONDA_BUILD}" = "1" ]]
```

Comment:
Note to self: the extra backslash is needed because this is part of codegen, not the actual script.

Reply:
To be eliminated in #91.